pkgs/top-level/all-packages.nix

```diff
@@ -4758,6 +4758,8 @@ with pkgs;
 
   spacevim = callPackage ../applications/editors/spacevim { };
 
+  srvc = callPackage ../development/tools/srvc { };
```

Comment:
You should put it on `applications/version-management/`.

Reply:
Moved thither, thanks